changelog.txt

```diff
@@ -1,3 +1,25 @@
+4.15
+============
+New Features:
+
+**MSAL has been upgraded to use Andorid X** MSAL.NET will now use the latest Android SDKs for it's Xamarin.Android platform. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1653)
+
+**GetAccounts can now filter by user flow for B2C accounts** MSAL's GetAccounts() api will now allow you to pass in a user flow to filter B2C accounts by when quering the cache. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1889)
+
+Bug Fixes:
+
+**MSAL can now migrate from ADALV3 to MSALV3 when multiple resourceId's are used** MSAL will now  ignore ADAL resource strings when fetching RT to enable migration from ADALV3 to MSALV3 cachetokens. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1815)
+
+**MSAL msaintains correlation id in broker flows** MSAL will now maintain the correlation ID of the authentication request throughout it's entire execution. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1805)
+
+**MSAL now contains the correct value for "ExpiresOn"** MSAL will now return the correct value for "ExpiresOn"  in the autentication result during brokered authenticaiton. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1806)
+
+**MSAL now has improved logic for AcquireTokenSilent(), GetAccounts() and RemoveAccount() during brokered authentication** During brokered authentication, MSAL will now check it's local cache for tokens first before sending the silent authenticaiton request to broker. GetAccounts will now merge the accouns from the local and broker caches when when returning results. RemoveAccount will now remove the account from both the local cache and the broker cache. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1820)
+
+**MSAL now has Better error reporting during Integrated Windows Authenticaiton** MSAL will now return the error body on WsTrust parse errors [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1838)
+
+**MSAL will now handle null intents returned to SetAuthenticationContinuationEventArgs** MSAL will now handle null intens returned to SetAuthenticationContinuationEventArgs to avoid throwing null reference exceptions. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/636)
```

Comment:
```suggestion
**MSAL will now handle null intents returned to `SetAuthenticationContinuationEventArgs`**. MSAL will now handle null intents returned to `SetAuthenticationContinuationEventArgs` to avoid throwing null reference exceptions. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/636)
```